src/main/java/org/spongepowered/api/command/spec/CommandSpec.java

```diff
@@ -254,7 +291,7 @@ public CommandSpec build() {
             }
 
             return new CommandSpec(this.args, this.executor, this.description, this.extendedDescription, this.permission,
```

Comment:
Yeah I did think about this (also this PR will likely end up being inplemented differently) but I failed to find an elegant solution and in the end I decided to keep it simple as to not have to hold the developers hand through this.